deep-security-agent/tasks/activate-linux.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2018, Trend Micro
+#
+# License as per [repo](master/LICENSE)
+#
+# *********************************************************************
+# * Activate the Deep Security Agent (For Linux only)
+# *********************************************************************
+---
+# *********************************************************************
+# * Check if DSA installed properly
+# *********************************************************************
+- debug:
+    msg: "Check if DSA installed properly."
+
+- stat:
+    path: /opt/ds_agent/dsa_query
+  register: dsaQuery
+
+- fail:
+    msg: 
+        - "Could not find command /opt/ds_agent/dsa_query to query DSA certificate."
+        - "Please make sure Deep Security Agent has been installed"
+  when: dsaQuery.stat.size is not defined 
+
+# *********************************************************************
+# * Check if DSA activated already
+# *********************************************************************
+- debug:
+    msg: "Check if DSA activated already."
+
+- set_fact:
+    isActivated: false
+
+- shell: /opt/ds_agent/dsa_query -c GetAgentStatus agentCertHash
+  register: certHash
+  become: true
+
+- set_fact:
+    isActivated: true
+  when: (certHash.stdout | regex_search('(D0:68:DE:48:1B:DC:4D:00:89:07:7F:85:15:20:C5:90:11:EB:56:6A)')) != "D0:68:DE:48:1B:DC:4D:00:89:07:7F:85:15:20:C5:90:11:EB:56:6A"
+
+- debug:
+    msg: "isActivated: {{ isActivated }}"
+
+# *********************************************************************
+# * Construct dsa_control arguments
+# *********************************************************************
+- debug:
+    msg: "Construct dsa_control arguments."
+
+- set_fact:
+    dsa_contrl_argument: "dsm://{{ dsa_activation_hostname }}:{{ dsa_activation_port }}/"
+    
+- set_fact:
+    dsa_contrl_argument: "{{ dsa_contrl_argument }} 'tenantID:{{ tenant_id }}' 'token:{{ token }}'"
+  when: >
+        tenant_id != none and 
+        token != none
+
+- set_fact:
+    dsa_contrl_argument: "{{ dsa_contrl_argument }} 'policyid:{{ policy_id }}'"
+  when: policy_id != none   
+
+- debug:
+    msg: "Running dsa_control with args: {{ dsa_contrl_argument }}"
```

Comment:
Not sure if it is the best idea to show the whole argument. token seems like a privacy to some customer.
I suggest to remove it or at least we need to have a (default) way to avoid print it.

Reply:
Removed displaying "tenant_id" and "token". If customer encounter some problem we may ask them to run with 
```
ansible-playbook default.yml -v
```
to get the tenant_id and token for debug.